fs2-aws-s3/src/main/scala/fs2/aws/s3/S3.scala

```diff
@@ -328,15 +324,12 @@ object S3 {
             .last
             .flatMap {
               case Some(resp) =>
-                Pull.eval {
-                  Async[F].blocking {
-                    val bs  = resp.asByteArray()
-                    val len = bs.length
-                    if (len < 0) None else Some(Chunk(unsafeWrapArray(bs)*))
```

Comment:
Also, why are we checking to see if the array size is negative? What would that even mean? This SO post says that can't happen: https://stackoverflow.com/a/11530038/283260
Should this just always return `Some`?

Reply:
Pushed a second commit removing this length check.